.circleci/config.yml

```diff
@@ -34,7 +46,7 @@ shared:
   # Commmon environment variables
   common_environment: &common_environment
     QT5: True
-    CMAKE_OPTS: -DUSE_WERROR=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo -DUSE_CCACHE=ON
+    CMAKE_OPTS: -DCMAKE_BUILD_TYPE=RelWithDebInfo -DUSE_CCACHE=ON
```

Comment:
I think removing `-DUSE_WERROR=ON` from Linux/Windows builds is not good. Can we leave this as-is and override it on the Mac section?

Reply:
You're right.